.github/workflows/build.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Build wheels
         uses: pypa/cibuildwheel@v3.1.2
         env:
-          CIBW_SKIP: "pp* *-musllinux* *-muslinux* *-win32"
+          CIBW_SKIP: "pp* *-musllinux* *-muslinux* *-win32 cp31?t-*"
```

Comment:
Could you walk us through the syntax here? Is `?` acting as a wild card? I looked at https://cibuildwheel.pypa.io/en/stable/options/ briefly